<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kierto</class>
 <widget class="QWidget" name="Kierto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="polkuCombo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="siirraNappi">
             <property name="text">
              <string>Siirrä</string>
             </property>
             <property name="icon">
              <iconset resource="../pic/pic.qrc">
               <normaloff>:/pic/siirra.png</normaloff>:/pic/siirra.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="ibanLabel">
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="kopioiIban">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../pic/pic.qrc">
               <normaloff>:/pic/edit-copy.png</normaloff>:/pic/edit-copy.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="barCopyButton">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../pic/pic.qrc">
               <normaloff>:/pic/bar-copy.png</normaloff>:/pic/bar-copy.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="viivakoodiLabel">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>35</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,0,1">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Vastaanotto</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="kuva1">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../pic/pic.qrc">:/pixaby/skanneri.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="vCheck">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="vInfo">
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="vAloita">
            <property name="text">
             <string>Aloita kierto</string>
            </property>
            <property name="icon">
             <iconset resource="../pic/pic.qrc">
              <normaloff>:/pic/seuraava.png</normaloff>:/pic/seuraava.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../pic/pic.qrc">:/pixaby/oikealle.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Tarkastus</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="kuva2">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../pic/pic.qrc">:/pixaby/tarkastus.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tCheck">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tHylatty">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../pic/pic.qrc">:/pic/peru.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tInfo">
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="tOk">
            <property name="text">
             <string>Tarkasta</string>
            </property>
            <property name="icon">
             <iconset resource="../pic/pic.qrc">
              <normaloff>:/pic/ok.png</normaloff>:/pic/ok.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tHylkaa">
            <property name="text">
             <string>Hylkää</string>
            </property>
            <property name="icon">
             <iconset resource="../pic/pic.qrc">
              <normaloff>:/pic/peru.png</normaloff>:/pic/peru.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../pic/pic.qrc">:/pixaby/oikealle.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Hyväksyntä</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="kuva3">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../pic/pic.qrc">:/pixaby/leimasin.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hCheck">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hHylatty">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../pic/pic.qrc">:/pic/peru.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="hInfo">
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="hOk">
            <property name="text">
             <string>Hyväksy</string>
            </property>
            <property name="icon">
             <iconset resource="../pic/pic.qrc">
              <normaloff>:/pic/ok.png</normaloff>:/pic/ok.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hHylkaa">
            <property name="text">
             <string>Hylkää</string>
            </property>
            <property name="icon">
             <iconset resource="../pic/pic.qrc">
              <normaloff>:/pic/peru.png</normaloff>:/pic/peru.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>36</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QSvgWidget" name="svgWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header>QSvgWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
